app/views/phases/_sort_and_filter.html.erb

```diff
@@ -0,0 +1,59 @@
+<div class="grid-container padding-x-0">
+  <div class="grid-row">
+    <div class="grid-col-12 display-flex flex-column flex-align-start padding-top-105">
+      <nav aria-label="Sort and filter options" class="usa-nav" data-controller="sort-filter-menu">
```

Comment:
the nav is hidden below 1024 because of a style on the `usa-nav` class but the figma designs show this dropdown available on mobile.